### PR TITLE
update backend initial Readme.md

### PR DESCRIPTION
**Pull request by TomatoesBurner**
ReadMe.md

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -1,9 +1,71 @@
...
+
+- **`controllers/`**: Controllers contain the logic for handling API requests and responses. For instance, they process data, interact with models, and return responses to the client.
+
+- **`coverage/`**: This folder is generated by your test coverage tool (like Jest). It contains reports that show how much of your code is covered by the tests.
+
+- **`models/`**: This folder includes the database models. These are Mongoose schemas that define the structure and data types of your MongoDB collections.
+
+- **`routers/`**: Routers define the routes for the API. They link specific API endpoints (e.g., `/api/v1/users`) to the corresponding controllers.
+
+- **`types/`**: If you are using TypeScript, this folder would store type definitions, which help in type-checking and enforcing data structure.
```
Comment (thogb) [inline backend/README.md:53]: Need proof read and update parts like this.

File: backend/README.md [hunk 2]
```diff
@@ -1,9 +1,71 @@
...
+
+- **`utils/`**: Utility functions that are used across different parts of the project, such as functions for error handling, token generation, or other helper logic.
+
+- **`.gitignore`**: Specifies which files and directories Git should ignore (e.g., `node_modules/`, `config.env`).
+
+- **`.prettierrc`**: Prettier configuration file for consistent code formatting across the project.
+
+- **`app.js`**: This is the main application file where the Express server is configured. It typically initializes middleware, connects to the database, and sets up the routes.
+
+- **`config.env`**: This file contains environment-specific variables like `MONGO_URI`, `JWT_SECRET`, or `AWS_S3_BUCKET`. It’s ignored by Git to avoid leaking sensitive information.
```
Comment (thogb) [inline backend/README.md:63]: It is not ignored at the moment.

Comment (TomatoesBurner) [reply backend/README.md:63]: > It is not ignored at the moment.

OK I will fix it.

File: backend/README.md (hunk 1 above)
Comment (TomatoesBurner) [reply backend/README.md:53]: > Need proof read and update parts like this. All "you", "your" should be removed.

get it.I will remove it.

**Pull request merged**